backend/src/ipfs.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from "express";
+import * as dotenv from "dotenv";
+import { getStarknetId } from "./starknet";
+
+dotenv.config();
+
+interface Proposal {
+  text: string;
+  address: string;
+  starknet_id?: string;
+}
+
+const ADDRES_REGEX: RegExp = /^0x0[0-9a-fA-F]{63}$/;
```

Comment:
this will match only addresses starting with 0x0 no?

Address can be even 0x1234

Also, not necessarily 63

Reply:
Alright, will fix it right away

Reply:
63 exactly is fine I think, because we can pad with zeros

Reply:
should be 64 actually, 0x is two, 64 is the rest

Reply:
okay